mkdocs.yml

```diff
@@ -73,6 +73,9 @@ pages:
   # API Changes: api/changes.md                 # page-tree = NEED_NEW_PAGE
 - Hooks:
   - Using hooks: hooks.md                        # page-tree = NEED_PAGE_MOVE to /hooks/usage.md
```

Comment:
@seanmadsen totally agree. tbh I'm not totally satisfied with the structure of the hooks page. It worked better in the wiki when you could see an overview with all hooks listed, but in the new docs you don't. You can still search for one if you know the name, but for a beginner coming in there's no top level overview, just a lot of hook category pages (Batch Hooks, Case Hooks etc..) and from my newbie perspective I'm not sure of the meaning. For example, (coming from Symfony) I might think for some entity change hook I might look under "Entity Hooks" but that's not what I want. 

How about a single folder for all hooks with a page listing all of them by category [like this one](https://api.drupal.org/api/drupal/includes%21module.inc/group/hooks/7.x)? The downside is that would create two places that need to be changed unless there are a way of autogenerating this page. Or maybe someone has a better suggestion.

Regarding the setup page, right now it's just these two pages. We might want a separate folder for each framework. Do you think there'll be many other framework-specific stuff in the other sections? If so it might make sense to keep this inside the hooks folder and merging them into a single "Setting up hooks" page, which would look like

```
# top level
- hooks.md
- hooks:
  - Framework Configuration: /hooks/configuration.md
  - Hook Reference: /hooks/reference.md
  - reference:
    - hook_civicrm_batchItems: hooks/reference/hook_civicrm_batchItems.md
    - all other hook files...
```